loadtest/scripts/metrics.py

```diff
@@ -48,6 +75,16 @@ def get_metrics():
     total_txs_num = sum([block["number_of_txs"] for block in skip_edge_blocks])
     average_txs_num = total_txs_num / len(skip_edge_blocks)
 
+    # Best block stats:
```

Comment:
the definition of "Best Block" I used was actually the block with the highest throughput, not necessarily the most transactions. I wonder if it'd be easier to just print a tuple (number of tx, tx per second, duration, tx mapping) for every block